From 795e93b858092b2940dca841477cee5a69caad4b Mon Sep 17 00:00:00 2001
From: alain kouassu <alain.kouassu@intel.com>
Date: Thu, 3 Jan 2013 15:40:37 +0100
Subject: [PORT FROM MAIN] ueventd: provide right access to tty for telephony components

BZ: 77948

provide right access to tty for telephony components.
This is needed for at_proxy.

Change-Id: I7d31510b708e4d394533db359c1c652b3cb10b6b
Orig-Change-Id: I5a5e5f1018e23aab4b6430b1889f568a7bec2d26
Signed-off-by: alain kouassu <alain.kouassu@intel.com>
Reviewed-on: http://android.intel.com:8080/84725
Reviewed-by: Continente, Christophe <christophe.continente@intel.com>
Reviewed-by: Butscher, MorganeX <morganex.butscher@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Lucas, Guillaume <guillaume.lucas@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 rootdir/ueventd.rc |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 49f6d3b..d1f58b1 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -79,6 +79,9 @@
 /dev/mdmTrace             0660   root       radio
 /dev/ttyACM*              0660   root       radio
 /dev/ttyPTI1              0660   root       radio
+/dev/gsmtty*              0660   root       radio
+/dev/ttyMFD*              0660   root       radio
+/dev/ttyGS0               0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

